docker-compose.yml

```diff
@@ -1,5 +1,7 @@
 services:
   api:
+    profiles:
+    - api
```

Comment:
Is the purpose of adding this so we can run just the api in one command like _docker-compose --profile api up_ or something to that effect?

Reply:
It's so that you can launch the DB and liquibase separately if you want to so that you can run the API from your IDE for debugging, etc